test/gravity.sql

```diff
@@ -0,0 +1,141 @@
+CREATE TABLE whitelist
```

Comment:
We may want to start this file with
```suggestion
PRAGMA FOREIGN_KEYS=ON;

CREATE TABLE whitelist
```

Reply:
I will add it for the purpose of local testing, but note that foreign keys are automatically turned on as part of the connection setup (see [`SqliteFKConnectionManager`]).

[`SqliteFKConnectionManager`]: https://github.com/pi-hole/api/blob/0a3f2be3751d4769d2778e907195bfb4462f2e24/src/databases/foreign_key_connection.rs#L62-L64